SaveOptions type is missing hooks option

<!--
If you don't follow the issue template, your issue may be closed.
Please note this is an issue tracker, not a support forum.
For general questions, please use StackOverflow or Slack.
-->

## Issue Description

### What are you doing?

<!--
We have a repository dedicated to make it easy for you to create an SSCCE.
https://github.com/papb/sequelize-sscce
Please consider using it, everyone wins!
-->

<!--
If you don't want to use the SSCCE repository, you can also post a MINIMAL, SELF-CONTAINED code that reproduces the issue. It must be runnable by simply copying and pasting into an isolated JS file, except possibly for the database connection configuration.
Check http://sscce.org/ or https://stackoverflow.com/help/minimal-reproducible-example to learn more about SSCCE/MCVE/reprex.
-->

This TypeScript fails a type check in V6 (given a model is defined)

```js
  const user = await User.findByPk(id);
  if (user === null) {
    return;
  }
  user.lastLogin = new Date();
  await user.save({ hooks: false });
```

The typing for SaveOptions is missing `hooks?: boolean` in it. Looking at the code, it appears the `hooks` option is still there, and it is documented, so this is a type error I think.

### What do you expect to happen?

<!-- Explain what behavior you wanted/expected. You may include an output. -->

The snippet above should pass a typecheck.

### What is actually happening?

<!-- Show what happened. You can skip this part if you included a link to an SSCCE above. -->

```
error TS2345: Argument of type '{ hooks: boolean; }' is not assignable to parameter of type 'SaveOptions<any>'.
```

### Additional context

Add any other context or screenshots about the feature request here.

### Environment

- Sequelize version: 6.2.3
- Node.js version: v14.1.0
- Operating System: macOS
- If TypeScript related: TypeScript version: 3.9.5

## Issue Template Checklist

<!-- Please answer the questions below. If you don't, your issue may be closed. -->

### How does this problem relate to dialects?

<!-- Choose one. -->

- [x] I think this problem happens regardless of the dialect.
- [ ] I think this problem happens only for the following dialect(s): <!-- Put dialect(s) here -->
- [ ] I don't know, I was using PUT-YOUR-DIALECT-HERE, with connector library version XXX and database version XXX

### Would you be willing to resolve this issue by submitting a Pull Request?

<!-- Remember that first contributors are welcome! -->

- [ ] Yes, I have the time and I know how to start.
- [x] Yes, I have the time but I don't know how to start, I would need guidance. (specifically I don't know where the types are actually defined, I can't see them in this repo)
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

--- a/types/lib/model.d.ts
+++ b/types/lib/model.d.ts
@@ -964,7 +964,7 @@ export interface SetOptions {
 /**
  * Options used for Instance.save method
  */
-export interface SaveOptions<TAttributes = any> extends Logging, Transactionable, Silent {
+export interface SaveOptions<TAttributes = any> extends Logging, Transactionable, Silent, Hookable {
   /**
    * An optional array of strings, representing database columns. If fields is provided, only those columns
    * will be validated and saved.

